autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Please spell out each acronym on the first use (ie <code>auto-scaling group (ASG)</code>).</p><p> </p><p>For hyperlinks, use the page title as the link text, rather than "here", for accessibility - suggested <code>You can view deployed ASGs at [EC2 Auto Scaling] in the console.</code>	 Also note that this link directs to us-east-1 and customers may have deployed ASGs in a different region.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>

Reply:
<p>also stripped region out of the link</p>